graphs/energy/nodes/industry/industry_heat_engine_gas_power_fuelmix.ad

```diff
@@ -0,0 +1,31 @@
+# this node is activated once input for external_coupling_capacity_of_industry_chp_combined_cycle_gas_power_fuelmix is set
+# this node provides the heat output of the connected chp
+# full_load_hours are an estimation for the must-run characteristic of heat supply from industrial CHPs
+# steam hot water efficiency is a generic value which will be updated when external coupling is activated
+# cost attributes are set to 0 since these are allocated to the connected CHP
+
+- output.loss = elastic
+- output.steam_hot_water = 0.9
+- groups = [preset_demand, heat_production, wacc_proven_tech, costs_production_heat_plants, sankey_conversion_group]
+- use = energetic
+- presentation_group = traditional_heat
```

Comment:
Note that this attribute is only relevant when the technical & financial properties of the node are shown in the frontend through the `related_node` attribute on ETModel.